src/Sensor/SensorForce.cpp

```diff
@@ -0,0 +1,34 @@
+#include "SensorForce.h"
+#define READ_INTERVAL 1000
+#define MIN_READ_VALUE 103
+#define MAX_READ_VALUE 977
+
+        SensorForce::SensorForce(pin_size_t pinNumber, PinMode mode) {
+            _pinNumber = pinNumber;
+            _pinMode = mode;
+        }
+
+        SensorForce::~SensorForce() { }
+        
+        String SensorForce::getHumanName() {
+            Serial.println(String("Force Sensor"));
+        }
+
+        void SensorForce::begin() {
+            pinMode(_pinNumber, _pinMode);
+        }
+
+        void SensorForce::handle() {
+            if(millis() > _lastReadTime + READ_INTERVAL) {
+                _lastReadTime = millis();
+                int int_force = analogRead(_pinNumber);
+                Serial.println(String(int_force));
```

Comment:
If you want to print debug info, then we can make a macro later for that later.

Otherwise, no need to call Serial.println in this method

Reply:
@underscoreHasan this was a pretty small change so I just went and made it myself.

PR approved.  I'll let you do the honours with merging it.